index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <nav id="navbar">
+      <a href="#welcome-section">About</a>
+      <a href="#projects">Work</a>
+      <a href="#contact">Contact</a>
+    </nav>
+
+    <section id="welcome-section">
+      <h1>Hey I am Mimic</h1>
+      <p>a web developer</p>
+    </section>
+
+    <section id="projects">
+      <h2 class="projects_title">These are some of my projects</h2>
+      <div class="projects-container">
+        <div class="project-tile">
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg"/>
+          <p>Map data accross the globe</p>
+          <a href="github.com" target="_blank">View  Project</a>
+        </div>
+
+        <div class="project-tile">
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg"/>
+          <p>Map data accross the globe</p>
+          <a href="github.com" target="_blank">View  Project</a>
+        </div>
+
+        <div class="project-tile">
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg"/>
+          <p>Map data accross the globe</p>
+          <a href="github.com" target="_blank">View  Project</a>
+        </div>
+
+      </div>
+      <a href="github.com" target="_blank" id="profile-link">Show All</a>
+    </section>
+
+    <section id="contact">
+      <h1>Let's work together...</h1>
```

Comment:
I think this should use h2-h6 because h1 is already used for "hey, I'm mimic" in the header

Reply:
It would be super helpful if you always give an explanation for your requirements. One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🤓 Kindly explain in your comments the reason for your changes 😎 

In this particular instance, you could explain why there should only be a `h1` tag on every HTML document 📝

Reply:
Edit:
I think you should use h2-h6 because h1 has already been used for "hey, I'm mimic" in the header. 
The reason behind this is h1 refers to the main titles of the page. And the page should have only 1 title to make it easier for search engines to index it. Each page should have its own unique title with h1 tag.